jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -140,8 +154,15 @@ public BaseImageWithAuthorization call()
                     registryCredential.getUsername(), registryCredential.getPassword());
 
         try {
-          return new BaseImageWithAuthorization(
-              pullBaseImage(registryAuthorization), registryAuthorization);
+          BaseImageWithAuthorization baseImageWithAuthorization =
+              new BaseImageWithAuthorization(
+                  pullBaseImage(registryAuthorization), registryAuthorization);
+
+          buildConfiguration
+              .getEventDispatcher()
+              .dispatch(new ProgressEvent(progressAllocation, 1));
```

Comment:
I think this should be `2` here. At this point, you always return successfully. `finally` may work?

Reply:
`RetrieveRegistryCredentialsStep` takes up another unit.